tests/test_api.py

```diff
@@ -330,9 +330,14 @@ def test_do_GET(self):
                                           'tag_key3=1&tag_key4=1.4&tag_key5=RandomTag&tag_key6=RandomTag2')
 
     def test_do_OPTIONS(self):
+        if MockPatroni.dcs.cluster:
+           MockPatroni.dcs.cluster.status.last_lsn = 20
```

Comment:
A better way of handling it.
```suggestion
    @patch.object(MockPatroni, 'dcs')
    def test_do_OPTIONS(self, mock_dcs):
        mock_dcs.cluster.status.last_lsn = 20
```